Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -166,9 +168,33 @@ class JobManagerExample: TableViewController {
         // But since this sample view controller can be pushed/pop, we need this.
         JobManager.shared.pauseAllJobs()
         
+        // clear out background tasks that we started for the jobs
+        backgroundTaskIdentifiers.forEach { UIApplication.shared.endBackgroundTask($0) }
+        backgroundTaskIdentifiers.removeAll()
+        
         super.viewWillDisappear(animated)
     }
     
+    deinit {
+        // clear out background tasks that we started for the jobs
+        backgroundTaskIdentifiers.forEach { UIApplication.shared.endBackgroundTask($0) }
+        backgroundTaskIdentifiers.removeAll()
```

Comment:
Isn't this unnecessary since the object is about to be deallocated?

Reply:
yep. Removing